canopen_motor_node/src/interface_mapping.h

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
It's not. The header is placed into the src folder, not the include folder.
I've used that method for private headers in some packages and it works great. However I haven't seen that in the ros_canopen packages, yet. It might not fit into the coding guidelines.